src/adapter/mysql.cr

```diff
@@ -91,12 +91,20 @@ class Granite::Adapter::Mysql < Granite::Adapter::Base
 
   # This will delete a row from the database.
   def delete(table_name, primary_name, value)
-    statement = "DELETE FROM #{table_name} WHERE #{primary_name}=?"
+    statement = "DELETE FROM #{quote_table_name(table_name)} WHERE #{quote_column_name(primary_name)}=?"
 
     log statement, value
 
     open do |db|
       db.exec statement, value
     end
   end
+
+  def quote_table_name(name : String) : String
+    %Q(`#{name.gsub("`", "``")}`)
```

Comment:
Simply for the purpose of more consistency between adapters, this could follow the same pattern as postgres.